packages/theme-product-form/README.md

```diff
@@ -30,11 +30,15 @@ and then import the functions you wish to use through ES6 imports:
 import {ProductForm} from '@shopify/theme-product-form'
 ```
 
+_NOTE: This package has a [`peerDependency`](https://yarnpkg.com/en/docs/dependency-types#toc-peerdependencies) on [`@shopify/theme-product`](https://github.com/Shopify/theme-scripts/blob/master/packages/theme-product/README.md)._
```

Comment:
```suggestion
_NOTE: This package has a [`peerDependency`](https://yarnpkg.com/en/docs/dependency-types#toc-peerdependencies) on [`@shopify/theme-product`](https://github.com/Shopify/theme-scripts/blob/master/packages/theme-product/README.md). You need to manually include [`@shopify/theme-product`](https://github.com/Shopify/theme-scripts/blob/master/packages/theme-product/README.md) to your project before `@shopify/theme-product-form`._
```

Reply:
Because the order doesn't matter in installing dependencies in `npm`, I would leave the messaging the same here but change the messaging below to reference `theme-product.js` and `theme-product-form.js` instead.